.github/workflows/mathesar.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
+    "folders": [
+        {
+            "path": "../.."
+        }
+    ]
+}
```

Comment:
Why is this added?

Reply:
oops sorry looks like my workspace got added by mistake. I will remove it

Reply:
I will also remove the unused imports